lib/Twig/RuntimeLoader.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class Twig_RuntimeLoader implements \Twig_RuntimeLoaderInterface
+{
+    private $mapping;
+
+    public function __construct(array $mapping = array())
+    {
+        $this->mapping = $mapping;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load($class)
+    {
+        if (isset($this->mapping[$class])) {
+            return $this->mapping[$class];
+        }
+
+        throw new \InvalidArgumentException(sprintf('Class "%s" is not mapped as Twig runtime.', $class));
```

Comment:
It must return null when it does not support the class according to the interface